source/voice_control/contribute-voice.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Contribute to the Voice initiative
+related:
+  - docs: /voice_control/best-practices
+    title: Best Practices with Assist
+  - docs: /voice_control/builtin_sentences
+    title: Sentences starter kit
+  - url: https://www.nabucasa.com/config/
+    title: Home Assistant Cloud
+---
+Home Assistant Voice relies on different technologies, models and data sources to be able to process wake words and commands in all languages. These projects, by ours and other communities, rely on volunteers to donate their time, voice or language knowledge for improvement.
+
+Note that models are trained on lots of data and that it can take time before you are able to see the impact of them.
```

Comment:
```suggestion
Note that models are trained on lots of data and that it can take time before you are able to see the impact of your contributions.
```